src/python/WMComponent/JobCreator/JobCreatorPoller.py

```diff
@@ -511,7 +513,7 @@ def pollSubscriptions(self):
 
             logging.debug("Going to call wmbsJobFactory for sub %i with limit %i", subscriptionID, self.limit)
 
-            splitParams = retrieveJobSplitParams(wmWorkload, workflow.task)
+            splitParams = retrieveJobSplitParams(wmWorkload, workflow.task, self.allowCreationFailure)
             logging.debug("Split Params: %s", splitParams)
```

Comment:
Given what I said above, I think we can remove this third parameter and add a line like
`splitParams.update(allowCreationFailure=self.allowCreationFailure)`
or something like that.

Reply:
See above. I would prefer to be able to override this per job splitter.